src/gui/Src/Gui/CPUDisassembly.h

```diff
@@ -169,6 +170,7 @@ public slots:
     MenuBuilder* mMenuBuilder;
     MenuBuilder* mHighlightMenuBuilder;
     bool mHighlightContextMenu = false;
+    const char *clipMsg = "The data has been copied to clipboard!\r\n";

```

Comment:
For me the copy notification has no use but you could add it in Bridge::CopyToClipboard I guess...